Sources/StreamChatUI/Utils/UILabel+Extensions.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+extension UILabel {
+    var adjustingFontForContentSizeCategory: Self {
+        adjustsFontForContentSizeCategory = true
+        return self
+    }
```

Comment:
```suggestion
    var withAdjustingFontForContentSizeCategory: Self {
        adjustsFontForContentSizeCategory = true
        return self
    }
```
Maybe this follows the convention better?

Reply:
Fixed. thanks :)